### PR TITLE
UI Tests Groundwork

### PR DESCRIPTION
**Pull request by heytherewill**
## What's this?
This fixes our existing UI tests so that they are up-to-date with the current designs

## Why do we want this?
Now that @ApocalypticFish can run tests on his machine, it'd be wise to have our Unit tests in great shape so he can both use them and start writing new ones!

## How is it done?
I'm basically updating the tests to consider new layouts and adding accessibility names to files. Easy peasy.

## Review hints
Review per commit and run the tests locally using VS

## :squid: Permissions
Me

### CODE REVIEW

File: Toggl.Tests.UI/Extensions/SignUpExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using Xamarin.UITest;
+
+namespace Toggl.Tests.UI.Extensions
+{
+    public static class SignUpExtensions
```
Comment (kornelijepetak) [inline Toggl.Tests.UI/Extensions/SignUpExtensions.cs]: I am a bit confused about the whitespace between statements in this file. Some tests have no blank lines between statements, some do. What is the convention we should use?

Comment (willsb) [reply Toggl.Tests.UI/Extensions/SignUpExtensions.cs]: I try grouping method calls to mimic actual user actions. I sometimes fail :P

**Comment by ApocalypticFish** [comment]
Test
TappingTheStopButtonStopsTheRunningTimeEntry
Is getting stuck because it is signing up with a new email, but it doesn't agree with the terms and conditions so gets stuck.

**Comment by ApocalypticFish** [comment]
Some tests are fixed, but others remain broken. Squashing this won't break anything in Develop that isn't already broken. Would like to squash this in favour of smaller branches/PR's for individual tests or batches of tests being fixed in future and so that future tests can be created from Develop.
Please squash if possible and I'll create some issues to fix the tests that aren't working and branch and create PR's for them as appropriate.

**Pull request merged**